_articles/short-courses.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: article
+author: julia
+title: "Learn how to work with data"
+date: 2017-06-09
+short: "OpenUp re-structures short course training offering"
+image: short-courses.jpg
+types:
+- name: launch
+themes:
+- name: education
+featured: true
+---
+
+We are excited to announce that we have re-shaped our data-driven storytelling [short course offering](https://openup.org.za/courses.html).  Our new structure of our curriculum provides a modular approach to the assembly of course material, allowing us to better address the unique skills of each of our learner groups, bringing more personal relevance to their learning experience.
+
+We now offer 5 short courses, that vary in length from 1 to 3 days, depending on the required modules.  Our courses are all centerred around data-driven storytelling and cover introductory, data sourcing, data cleaning, data analysing and data visualising content.
+
+We believe that an equal society starts with equal access to information, and to that end we promote the release of data under open data licences to make it available to everyone who wants and needs it.  To that end we are also involved in a number of data literacy programmes, with the aim of equipping South Africans with the skills, techniques, and tools that will enable them to become a community of data consumers.
+
+One of these data literacy programmes offers training in the form of short 1- to 3-day courses.  Our curriculum covers the different (often) sequential stages of the data processing pipeline, and has recently experienced an overhaul to provide a more concentrated focus on each of the stages of the Data Pipeline.
+
+[Check out our new course offering](https://openup.org.za/courses.html)and contact us at <training@openup.org.za> - we look forward to consulting with you on the training solution that would best suit your organisation’s needs.
```

Comment:
@longhotsummer just so you know putting email addresses between `<>` works as well